receiver/mongodbatlasreceiver/receiver.go

```diff
@@ -219,6 +236,8 @@ func (s *mongodbatlasreceiver) extractProcessDatabaseMetrics(
 		rb.SetMongodbAtlasProcessTypeName(process.TypeName)
 		rb.SetMongodbAtlasProcessID(process.ID)
 		rb.SetMongodbAtlasDbName(db.DatabaseName)
+		rb.SetMongodbAtlasRegionName(providerValues.RegionName)
```

Comment:
Not intending to scope creep should we probably add this attribute to some forms of logs that have access to the cluster ProviderSettings as well? 

https://github.com/open-telemetry/opentelemetry-collector-contrib/blob/f509060a8d1ab5ca4b5827e0c60d1149e3059908/receiver/mongodbatlasreceiver/access_logs.go#L307

https://github.com/open-telemetry/opentelemetry-collector-contrib/blob/ae6d36bc4001f539433e6b03fcff365d9489bf4d/receiver/mongodbatlasreceiver/logs.go#L151

Reply:
Got it, added the provider and region name to the logs and updated the tests accordingly. Good suggestions.

Reply:
@JonathanWamsley seems like some test failures?

Reply:
ahh, my bad.  I updated the mock and golden file for the access logs integration test.